src/Common/src/System/Collections/HashHelpers.cs

```diff
@@ -57,7 +57,7 @@ public static bool IsPrime(int candidate)
         public static int GetPrime(int min)
         {
             if (min < 0)
-                throw new ArgumentException(SR.Arg_HTCapacityOverflow);
+                throw new ArgumentException(SR.Arg_HTCapacityOverflow, nameof(min));
```

Comment:
Internal helper that does validation for public APIs. This change should be reverted.